### DISCUSSION TITLE
微积分 | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#51] [Announcements]
# 微积分 | HackWay技术学习路线

课程名称： Structure and Interpretation of Computer Programs

https://hackway.org/docs/math/book/calculus

**Comment by zou8944** [comment]
此处文不对题吧，明明是微积分，实际却是计算机程序的构造于解释？

**Reply by hackwayorg** [reply]
目前AI和数学部分内容还没有更新，先把cs相关内容更新了会来处理

**Reply by Ginkgolide** [reply]
大家知道怎么观看video吗，YouTube显示是私人视频
